Client/src/Components/Graph/Graph.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import * as d3 from "d3"
+
+//Global variables for graph creation
+
+var margin = {
+    top: 50,
+    right: 210,
+    bottom: 50,
+    left: 70
+},
+    outerWidth = 1050,
+    outerHeight = 500,
+    width = outerWidth - margin.left - margin.right,
+    height = outerHeight - margin.top - margin.bottom;
+
+var xScale, yScale, xAxis, yAxis;
+
+//This function creates the basis for the linear x and y axis
+function startLinear() {
```

Comment:
Similarly const function = () =>

Arrow functions are the way to go!

Reply:
done!